.gitpod.dockerfile

```diff
@@ -3,5 +3,61 @@ FROM gitpod/workspace-full
 USER root
 RUN sudo apt-get update && apt-get install -y apt-transport-https \
  && sudo apt-get install -y \
-    xserver-xorg-dev libxext-dev libxi-dev build-essential libxi-dev libglu1-mesa-dev libglew-dev pkg-config libglu1-mesa-dev freeglut3-dev mesa-common-dev \
+    xserver-xorg-dev \
```

Comment:
We need so many? :scream_cat: 
Gitpod already takes a very long time to start...

Reply:
the first time -- after that it should be able to re-use the prebuilt container. I got this list from docs here (sorry should have linked):

https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#chrome-headless-doesnt-launch-on-unix

if you find any we can skip, that'd be great, but it's pretty painstaking to try over and over! Thanks!